### PR TITLE
Create 003-use-emotion.md

### PR DESCRIPTION
**Pull request by davidflanagan**
@schalkneethling for your review. I know you also like Emotion, but let me know if you think I should add anything here.

@jwhitlock: for your approval

### CODE REVIEW

File: ADRs/003-use-emotion.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+styles as code (just as there are for expressing HTML layouts as
+code). Also, CSS-in-JS libraries can do things like automatically
+adding vendor prefixes to CSS properties when needed. Developers often
+find that CSS-in-JS improves their productivity. There are quite a few
+CSS-in-JS libraries in common use, and the point of this ADR is to
+settle on one.
+
+### Consequences
+
+While we will probably still use standard CSS stylesheets to define
```
[the remark this answers is not in the document]
Comment (davidflanagan) [reply ADRs/003-use-emotion.md]: I didn't mean to suggest that we wouldn't use a pre-processor to create those standard CSS stylesheets.  I'll clarifiy.

File: ADRs/003-use-emotion.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+with hyphens don't need to be converted to camelCase identifiers, and
+so that only a single string is involved instead of an object. Second
+it introduced the syntax of using tagged template literals to create
+new React components given an HTML tag name and a string of
+CSS. Unlike Aphrodite where you define a bunch of objects to represent
+a bunch of CSS classes, Styled Components allows you to define a bunch
+of simple React components that are efficiently pre-styled just how
+you want them.
+
+Styled Components is now one of the most popular css-in-js libraries.
```
[the remark this answers is not in the document]
Comment (davidflanagan) [reply ADRs/003-use-emotion.md]: I couldn't find current benchmarks on performance, and both libraries have been evolving so quickly that I didn't want to make a perf claim. I think I say something later about Emotion having the momentum and growth, but if not will add that.

**Comment by davidflanagan** [comment]
@jwhitlock: I've addressed the review comments on this one, so I think it is ready for approval.

**Comment by jwhitlock** [comment]
This feels less certain to me than the React choice. It is nice that [Emotion](https://github.com/emotion-js/emotion) and [Styled Components](https://www.styled-components.com) have a similar API, maybe this will be a mini-standard going forward. It feels like there is a strong chance we'll have to revisit this in 2 years, the ecosystem seems too immature. But, I guess the future devs can propose that change 😼

File: ADRs/003-use-emotion.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
+## Use Emotion as our CSS-in-JS solution
+
+|Status       | proposed <!--becomes accepted, rejected or superseded later-->|
+|-------------|---------------------------------------------------------------|
+|**Proposed** | 2019-02-04
+|**Accepted** | (the date the proposal was accepted/rejected)
```
Comment (jwhitlock) [inline ADRs/003-use-emotion.md]:
```suggestion
|**Accepted** | 2019-02-05
```

**Pull request merged**